networks/rpc/http.go

```diff
@@ -263,6 +263,11 @@ func NewFastHTTPServer(cors []string, vhosts []string, timeouts HTTPTimeouts, sr
 		handler = newNewRelicHTTPHandler(nrApp, handler)
 	}
 
+	ddTracer := newDatadogTracer()
```

Comment:
It looks that `newDatadogTracer` must return `DatadogTracer` object because of the name. When I see your implementation, datadog sdk doesn't provide any struct or interface. And so I suggest the below codes to follow  the existing newrelic implementation.

```golang
ddTracer := newDatadogTracer()
if ddTracer != nil {
    handler = newDatadogHTTPHandler(ddTracer, handler)
}
```

```golang
type DatadogTracer struct {
    Tags                    []string
     KlaytnResponse bool
}

func newDatadogTracer() *DatadogTracer {
	ddTraceEnabled := os.Getenv("DD_TRACE_ENABLED")
	if strings.ToLower(ddTraceEnabled) != "true" {
		return nil
	}
        
        tags := strings.Split(os.Getenv("DD_TRACE_HEADER_TAGS"), ",")
	klaytnResponse := strings.ToLower(os.Getenv("DD_KLAYTN_RPC_RESPONSE")) == "true"
	tracer.Start()

	return &DatadogTracer{tags, klaytnResponse}
}

func newDatadogHTTPHandler(ddTracer *DatadogTracer, handler http.Handler) http.Handler {
...
}
```

Reply:
It seems your method is more clear.
It will be reflected in PR.